Gemfile

```diff
@@ -24,6 +24,7 @@ gem 'faker'
 gem 'omniauth-google-oauth2'
 gem 'omniauth'
 gem 'dotenv-rails'
+gem 'seed_dump'
```

Comment:
This is an interesting gem that I'm not 100% sold on. you can call `rails db:seed:dump` and it will take the data in the local database and create a seed file out of it.